src/content/docs/browser/new-relic-browser/browser-apis/recordReplay.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: recordReplay
+type: apiDoc
+shortDescription: A method used to force a replay to start recording
+tags:
+  - Browser
+  - Browser monitoring
+  - Browser Agent
+metaDescription: A method used to force a replay to start recording
+redirects:
+freshnessValidatedDate: 2024-04-22
+---
+
+## Syntax
+
+```js
+newrelic.recordReplay()
+```
+
+Browser API used to force a replay to start recording.
+
+## Requirements
+
+* Browser Pro, or Pro+SPA agent (v1.240.0 or higher)
+
+## Description
+
+`newrelic.recordReplay()` can be called to manually force a replay to begin recording. All prerequisites of recording a replay must be met in order to succesfully trigger a replay to record.  This includes:
+* Account is entitled to record replays
+* Session Tracking is Enabled
+* Session Trace is Enabled 
+* Browser mutation observer global is present in the current version of the browser being used
+* The Session Replay feature is set to `autoStart: false` and has not been started
```

Comment:
```suggestion
* Session tracking is Enabled
* Session trace is Enabled 
* Browser mutation observer global is present in the current version of the browser being used
* The session replay feature is set to `autoStart: false` and has not been started
```

Reply:
@metal-messiah Are these settings added to the UI app settings page yet? If so, I'll add instructions to the session replay doc and link to them from here.

Reply:
The `session tracking` setting is in the UI.  The `session trace` and `autoStart` settings are not